docs/DeveloperGuide.md

```diff
@@ -323,11 +323,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. CaseTrack shows an error message.
 
       Use case resumes at step 2.
 
-*{More to be added}*
+
```

Comment:
It should be ok to leave this in as other team members will also be adding their Use Cases.